lib/authentication/view/sign_up_view.dart

```diff
@@ -45,25 +51,43 @@ class _SignUpViewState extends State<SignUpView> {
     _fetchFilter();
   }
 
+  void parse(TextEditingController email, TextEditingController firstName,
```

Comment:
Call it `parseNameFromEmail` so it's clearer what it does.

Reply:
Got it!